config/environment.js

```diff
@@ -5,7 +5,8 @@ module.exports = function (environment) {
     modulePrefix: 'ember-website',
     environment,
     rootURL: '/',
-    locationType: 'auto',
+    locationType: 'trailing-history',
+    historySupportMiddleware: true,
```

Comment:
For more information, I think we can look at https://dockyard.com/blog/2019/12/12/ember-location-api-part-2-extending-and-customizing